usr/share/rear/backup/BLOCKCLONE/default/400_copy_disk_struct_files.sh

```diff
@@ -7,8 +7,8 @@ if [ -z "$BLOCKCLONE_SAVE_MBR_DEV" ]; then
     return
 fi
 
-local backup_path=$( url_path $BACKUP_URL )
-local opath=$(backup_path $scheme $path)
+local backup_path=$( url_path "$BACKUP_URL" )
+local opath=$( backup_path "$scheme" "$path" )
```

Comment:
If you don't quote the result of the expansion here and also `$opath` below, quoting `$path` here will not have much effect in many cases (although that's not directly related to the IPv6 problem, because the shell metacharacters don't appear in the path part of the URL).

I don't see where is `path` getting defined anyway, so for the cases where `${path}` is returned on output, the code has not worked anyway, thus quoting is not that a big issue.